templates/Dockerfile-deb.tmpl

```diff
@@ -31,7 +31,7 @@ RUN set -ex; \
     export GNUPGHOME="$(mktemp -d)"; \
 # Fix ipv6 issue on travis: https://github.com/f-secure-foundry/usbarmory-debian-base_image/issues/9#issuecomment-466594168
     echo "disable-ipv6" >> $GNUPGHOME/dirmngr.conf; \
-    gpg --batch --keyserver keyserver.ubuntu.com --recv-keys "$key"; \
+    gpg --batch --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "$key"; \
```

Comment:
In local development without protocol and port, firewall could block downloading gpg key. With this addition. keys are being downloaded from 80 port which is open for nearly all firewalls